treeopt/treeOpt.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import os
+
+#Import of treeopt submodules
+import modules.sampling as sampling
+import modules.simulate as simulate
+import modules.optimize as optimize
+import modules.benchmark as benchmark
+import modules.metamodell as metamodell
+import modules.visualize as visualize
+
+class TreeOpt:
+    """Python class, that bundles all modules for nessesary for adaptive black box metamodelling"""
+    
+    def __init__(self):
+        """Allocates variables with standard values"""
+        self.samplingMethod = sampling.latinHypercube
+        self.simulateMethod = simulate.simulateBenchmarkFunction
+        self.smMethod = metamodell.KRG
+    
+    #Functions for Data management
+    def appendXData(self, xi):
+        self.x = np.vstack([self.x, xi])
+        
+    def appendYData(self, yi):
+        self.y = np.vstack([self.y, yi])
+        
+    #Functions to safe and load data to/from file as csv
+    def writeData(self, npArray, filename):
+        """Function that writes a numpy Array into a file in the threeOptData direcory
+        INPUT:
+            array: numpy array to be written into the file
+            filename: name of the file
+        """
+        path = os.path.join(os.getcwd()+"/treeOptData",filename+".csv")
+        np.savetxt(path, npArray, delimiter=",")
+        
+    def readData(self, filename):
+        """Function that reads a file in the treeOptData directory and creates a numpy array with this data 
+        INPUT:
+            filename: name of the file to be read
+        OUTPUT:
+            numpy array containing the information of the file
+        """
+        path = os.path.join(os.getcwd()+"/treeOptData",filename+".csv")
+        array = np.loadtxt(path, delimiter=",")
+        return(array)
+        
+    #Set functions (to set functions/methods/variables prior to optimizaion)
+    def setName(self, name):
+        self.name = name
+    
+    def setSamplingMethod(self, method):
+        self.samplingMethod = method
+        
+    def setNumDOE(self, numDOE):
+        self.numDOE = numDOE
+        
+    def setLimits(self, limits):
+        self.limits = limits
+    
+    def setSimulateMethod(self, method):
+        self.simulateMethod = method
+        
+    def setBenchmarkingProblem(self, problem):
+        self.benchmarkingProblem = problem
```

Comment:
what does this do? Could you please add short docstrings to the member functions of the TreeOpt class?